cattle/plugins/docker/compute.py

```diff
@@ -278,6 +278,14 @@ def instance_activate(self, req=None, instanceHostMap=None,
 
         instance = instanceHostMap.instance
         host = instanceHostMap.host
+
+        try:
+            host.clusterConnection = \
+                instanceHostMap.data.fields['clusterConnection']
+            log.debug('clusterConnection = %', host.clusterConnection)
```

Comment:
since it's logged at debug, do you think that's ok?  i can even change it to trace too